### PR TITLE
README.md: Product description, features, installation

### LINKED ISSUE #89: Improve the readme

### PR DESCRIPTION
**Pull request by joswinter**
Added product description, features and installation to the README.md.
Should fix #89 Improve the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+- The **Graph Viewer** shows the contents of the graph in the structure of a tile lane diagram. It is able to visualize multiple DNA strands and the mutations between those DNA strands if a reference is selected. 
+- The **Tree Viewer** shows the contents of the phylogenetic tree in the structure of a Sunburst diagram. This shows the relation between the different samples used in the program.
+- The **Sequence Viewer** shows all sequences names so the user has an overview of the data which is currently viewed.
+
+
+## Features
+
+
+- View a single or multiple DNA strands as a graph to research the structure. In a unified form where every sequence segment has the same coordinate system and empty sequence segments are also visualized.
+- Create a phylogenetic tree based on several DNA strands so that the user can compare them and observe the timeline and the common ancestors of several functionalities.
```
Comment (jorenham) [inline README.md]: `Create` -> `Display`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+- Select the level of detail of the viewport of DNA so that the user can see the relevant parts of DNA using semantic zooming.
+
+
+
+
+## Installation
+
+The application can be executed by downloading or cloning the source. 
+
+To start researching your DNA sequence graph you need your files in the graph format which consists of two files with `.node.graph` and `.edge.graph` as extension.
```
Comment (jorenham) [inline README.md]: The `.node.graph` and `.edge.graph` files also need to have the same filename

**Comment by jorenham** [comment]
Looks nice, maybe something about the Context Project would look good

**Comment by albert-smit** [comment]
maybe add something about maven to the instalation instructions

**Comment by alangerak** [comment]
something about annotations would be nice too

**Comment by rutgervandenberg** [comment]
Looks good.

**Comment by joswinter** [comment]
What should we add about maven, I'm not an expert. We should wait with something about annotations until we know how it is going to be implemented.

**Pull request merged**